debugger/src/debug.c

```diff
@@ -1066,7 +1105,13 @@ void debug_init(void)
 	configfile = g_strconcat(geany_data->app->configdir, G_DIR_SEPARATOR_S, "geany.conf", NULL);
 	g_key_file_load_from_file(config, configfile, G_KEY_FILE_NONE, NULL);
 	font = utils_get_setting_string(config, "VTE", "font", "Monospace 10");
+#if GTK_CHECK_VERSION(3, 0, 0)
+	fontdesc = pango_font_description_from_string(font);
+	vte_terminal_set_font(VTE_TERMINAL(terminal), fontdesc);
+	pango_font_description_free(fontdesc);
+#else
```

Comment:
If you like you can replace this lines with a call to function ```gp_vtecompat_set_font_from_string()``` from the geany-plugins util lib. Requires ```#include <gp_vtecompat.h>```.

Reply:
I used the common utils library, thanks for the suggestion.